components/PartnerBenefits.tsx

```diff
@@ -0,0 +1,42 @@
+import Link from "next/link";
+
+interface IButtonProps {
+  title: string
+  linkPath: string
+}
+
+export default function PartnerBenefits () {
+
+  function LinkButton({title, linkPath}: IButtonProps) {
+    return (
+      <Link
+        className="bg-cesium-orange py-4 px-6 rounded-lg text-xl font-bold text-white"
+        href={linkPath}
+      >
+        {title}
+      </Link>
+    )
+  }
+
+  return (
+      <div className="relative text-center mt-6 h-[400px] bg-light-gray">
+        <h1 className="pt-8 font-bold text-xl">
+          Vantagens de ser Sócio
+        </h1>
+
+        <p className="flex justify-center pt-2 px-32">
```

Comment:
`<p className="flex justify-center opacity-70 pt-8 px-32">`

With these small changes I think it looks better

Reply:
thank you for the suggestion, i agree it looks better